src/modules/settings/components/Settings.jsx

```diff
@@ -1,15 +1,40 @@
 import React from 'react';
-// eslint-disable-next-line import/extensions, import/no-unresolved
-import './settings/*';
 import Panel from 'rsuite/Panel';
 import {Icon} from '@rsuite/icons';
 import InputGroup from 'rsuite/InputGroup';
 import AutoComplete from 'rsuite/AutoComplete';
 import * as faSearch from '@fortawesome/free-solid-svg-icons/faSearch';
-import {Components} from './Store.jsx';
 import FontAwesomeSvgIcon from '../../../common/components/FontAwesomeSvgIcon.jsx';
+import {loadModuleForPlatforms} from '../../../utils/modules.js';
+import {PlatformTypes} from '../../../constants.js';
 
-const settings = Object.values(Components).sort((a, b) => a.name.localeCompare(b.name));
+let settings = [];
+
+(async () => {
+  await loadModuleForPlatforms(
```

Comment:
rather than loading these here, load them from within the platform's settings file and pass in settings as a prop.

`twitch/Settings.jsx` and `youtube/Settings.jsx`